Snippets/Wcf/Wcf_1/Enum/Usage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus;
+
+namespace Wcf1.Enum
+{
+    class Usage
+    {
+        #region WcfEnum
```

Comment:
@SzymonPobiega leading and trailing whitespace in snippets is trimmed by the engine